ansible/files/airflow/runner.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+CMD="airflow"
+TRY_LOOP="10"
+POSTGRES_HOST=${DB_URI:-postgres}
+POSTGRES_PORT="5432"
+RABBITMQ_HOST="rabbitmq"
+RABBITMQ_CREDS="airflow:airflow"
+
+# Wait for RabbitMQ
+j=0
+while ! curl -sI -u $RABBITMQ_CREDS http://$RABBITMQ_HOST:15672/api/whoami |grep '200 OK'; do
+  j=`expr $j + 1`
+  if [ $j -ge $TRY_LOOP ]; then
+    echo "$(date) - $RABBITMQ_HOST still not reachable, giving up"
+    exit 1
+  fi
+  echo "$(date) - waiting for RabbitMQ... $j/$TRY_LOOP"
+  sleep 5
+done
+
+# Generate Fernet key for replacement below
+export DEFAULT_FERNET_KEY=$(python -c "from cryptography.fernet import Fernet; FERNET_KEY = Fernet.generate_key().decode(); print FERNET_KEY")
+export FERNET_KEY=${FERNET_KEY:-$DEFAULT_FERNET_KEY}
+
+export DEFAULT_DOCKER_API_VERSION=1.23
```

Comment:
It might be better to have it as an Airflow `Variable`, but for now I removed this env variable and the default is defined on the `helpers.py` as:

`docker_api_version = os.environ.get('DOCKER_API_VERSION', '1.23')`

That should be enough.